libs/components/popovers/testing/src/popover/harness/fixtures/popover-harness-test.component.html

```diff
@@ -0,0 +1,21 @@
+<div style="height: 200px"></div>
+<button
+  class="sky-btn sky-btn-default sky-margin-inline-compact"
+  type="button"
+  [skyPopover]="myPopover"
+  [skyPopoverAlignment]="popoverAlignment"
+  [skyPopoverPlacement]="popoverPlacement"
+  #directiveRef
+>
+  Open popover on click
+</button>
+
+<sky-popover
+  [dismissOnBlur]="dismissOnBlur"
+  [popoverTitle]="popoverTitle"
+  #myPopover
```

Comment:
How would the consumer target a specific popover in their test using `data-sky-id` (assuming they have multiple popovers on the same page)?

Reply:
Addressed with refactor